framework/src/modules/http_api/schema/swagger.yml

```diff
@@ -14,49 +14,49 @@
 swagger: '2.0'
 
 info:
-  title: Lisk API Documentation
+  title: Lisk API documentation
   description: |
     # Welcome!
 
-    ## Access Restrictions
-    The API endpoints are by default restricted to a whitelist of IPs that can be found in `config.json` in the section [`api.access.whitelist`](https://github.com/LiskHQ/lisk/blob/1.0.0/config.json#L35).
-    If you want your API to be accessable by the public, you can do this by changing `api.access.public` to `true`.
+    ## Access restrictions
+    The API endpoints are by default restricted to a whitelist of IPs that can be found in the Control API access section [`API Access`](https://lisk.io/documentation/lisk-core/3.0.0/management/api-access.html).
+    If it is required for your API to be accessible by the public, this can be achieved by changing `api.access.public` to `true`.
     This will allow anyone to make requests to your Lisk Core node.
-    However some endpoints stay private, that means only a list of whitelisted IPs can successfully make API calls to that particular endpoint;
-    This includes all forging related API calls.
-    By default, only the nodes' local IP is included in the whitelist, you can change the setting in `config.json` in the section [`forging.access.whitelist`](https://github.com/LiskHQ/lisk/blob/1.0.0/config.json#L114).
+    However some endpoints will stay private, hence this means that only a list of whitelisted IPs can successfully make API calls to that particular endpoint;
+    this also includes all forging related API calls.
+    By default, only the nodes' local IP is also included in the whitelist, nevertheless it is possible to change the setting in `config` in the API specification section [`forging status of a delegate`](https://lisk.io/documentation/lisk-sdk/reference/api.html#_description_11).
     For more details, see the descriptions at the respective endpoint.
 
     ## Requests
     Chained filter parameters are logically connected with `AND`.
     `HTTP` is the supported URL schema by default.
-    To enable `HTTPS`, please adjust the the [`ssl`](https://github.com/LiskHQ/lisk/blob/1.0.0/config.json#L124) section in `config.json`.
+    To enable `HTTPS`, please adjust the [`ssl`](https://lisk.io/documentation/lisk-sdk/reference/api.html#_requests) in the API specification section.
```

Comment:
Why not referring here to the management guide, for enabling SSL?

Reply:
```suggestion
    To enable `HTTPS`, please see the following link whereby it is possible to adjust the [`ssl`](https://lisk.io/documentation/lisk-core/management/ssl.html).
```

Reply:
Also I think we should amend line 28 to the following " In addition, for further details, see the descriptions at the respective endpoint''. (If you agree Mona?)